### PR TITLE
feat: options for peers keep-alive

### PR DESCRIPTION
**Pull request by simone-sanfratello**
I'd like to disable the `keep-alive` to check if the memory-leak is related to it, and if performance decrease

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -57,7 +59,9 @@ module.exports = {
   linkTableBlockKey: 'blockmultihash',
   linkTableCarKey: 'carpath',
 
+  enableKeepAlive: enableKeepAlive === 'true',
```
Comment (francardoso93) [inline src/config.js:62]: We should keep this consistent and easier to read

`enableKeepAlive: enableKeepAlive ?? 'true',`

**Review by francardoso93 — APPROVED** [review]
Minor comments and LGTM.

**Comment by francardoso93** [comment]
If we ended up finding out that's the issue, we also need to understand if there are side effects of letting keep alive disabled.

File: src/config.js (hunk 1 above)
Comment (simone-sanfratello) [reply src/config.js:62]: I'd like to release that and monitor for a day, then restore it
I think it's easier doing in this way, but happy with any solution

**Pull request merged**